source/fab/artefacts.py

```diff
@@ -32,7 +32,6 @@ def __call__(self, artefact_store):
             The artefact store from which to retrieve.
 
         """
-        pass
```

Comment:
Yes, can do - though that means that all derived classes (that atm do ` super().__call__(artefact_store)`) need to have that removed (and it's a pain to test ... since I can't instantiate a class if it doesn't implement `__call__`)

Reply:
Given that the abstract method doesn't do anything, apart from throw an error, it makes sense to not call them.